.travis.yml

```diff
@@ -1,11 +1,24 @@
-# Configuration script to Travis-CI
-sudo: false
+# Configuration script for Travis-CI
 language: python
 python:
     - 2.7
+matrix:
+    include:
+      - python: 2.7
+        name: "2.7"
+      - python: 2.7
+        name: "Flake8"
+        install:  pip install -U flake8
+        script: flake8 --doctests src setup.py
+        after_success:
```

Comment:
Do we need this empty `after_success` to avoid the call to coveralls?

Reply:
yes.